depthai_nodes/ml/messages/map.py

```diff
@@ -71,3 +72,16 @@ def height(self) -> int:
         @rtype: int
         """
         return self._height
+
+    def getVisualizationMessage(self) -> dai.ImgFrame:
+        """Returns default visualization message for 2D maps in the form of a
+        colormapped image."""
+        img_frame = dai.ImgFrame()
+        mask = self._map.copy()
+        if np.any(mask < 1):
```

Comment:
I think it would be safer to do min-max scaling here:
```
min_val = map.min()
    max_val = map.max()
    if map.min() != map.max():
        map = (map - min_val) / (max_val - min_val)
```
This way we always visualize a relative map.